inductiva/fluids/scenarios/fluid_tank/README.md

```diff
@@ -0,0 +1,44 @@
+### Fluid tank
```

Comment:
Maybe I wouldn't move it as deep as this. Maybe just build a longer README md where you place this and other CFD/SPH info. But: if you do move here (which is fine too), then promote this to H1

Reply:
Based on a previous @sarmento comment, I just thought it would fit nicely in the docs on the first level. Afterwards, they can be used for the ReadTheDocs...

But after merging all of the documentations a single person can still make that convergence or just leave it as it is.

Reply:
> But after merging all of the documentations a single person can still make that convergence or just leave it as it is.

Agreed.